package.json

```diff
@@ -48,16 +34,9 @@
         "stylelint-config-prettier": "^8.0.2",
         "stylelint-config-wordpress": "^17.0.0",
         "stylelint-prettier": "^1.1.2",
+        "stylelint-scss": "^3.19.0",
         "vue-template-compiler": "^2.6.12",
         "wp-pot": "^1.6.3",
         "wp-textdomain": "^1.0.1"
-    },
-    "dependencies": {
-        "npm": "^6.13.4",
-        "stylelint-scss": "^3.19.0"
-    },
-    "engines": {
-        "node": "8.9.1",
-        "npm": "5.5.1"
```

Comment:
These options actually conflict with some of the installed tools.

Reply:
It would be better to use something like [Volta](https://volta.sh) or fnm (or nvm, but beware it's notoriously slow and finnicky) to pin the projects Node tooling versions.